submodules/devices/devices.js

```diff
@@ -7,6 +7,10 @@ define(function(require) {
 		return _.every([dest, src], _.isArray) ? src : undefined;
 	};
 
+	var showTeammateDevice = monster.config.hasOwnProperty('extraDevices')
+		&& monster.config.extraDevices.length
+		&& $.inArray('teammate', monster.config.extraDevices) > -1;
+
```

Comment:
More declarative vs imperative:
```suggestion
	var showTeammateDevice = _
	    .chain(monster.config)
	    .get('extraDevices', [])
	    .includes('teammate')
	    .value();

```